package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "test": "mocha -r ts-node/register test/libraryUtilitiesTest.ts",
```

Comment:
Also, are those underscores required in folders like `__tests__` and `__mocks__`?